cibuildwheel/windows.py

```diff
@@ -227,6 +291,10 @@ def setup_python(
     else:
         assert_never(build_frontend)
 
+    if python_libs_base:
+        # Set up the environment for various backends to enable cross-compilation
```

Comment:
What do you mean "various backends" here? It's a setuptools-specific configuration, right?

Reply:
Right now only setuptools has an environment-based option for configuring it. Meson has a command-line option, but no environment option, and pymsbuild has both (but is not widely used, so isn't needed in cibuildwheel unless you really want it ;-) ).

I expect more backends in the future. Anyone who supports compiling native code eventually needs to either support some way to specify the target machine, or formally drop support for certain platforms. But we're in early "I don't want to go first" stage of this, so while they'll eventually have to come along, only setuptools is here right now.